src/make_data/banner_intervention/README.md

```diff
@@ -0,0 +1,113 @@
+ [Google Sheet document][SaBpages]) are presented with a banner that suggests that they might
+ want to visit the Start a Business checker.
+- The banner is tracked on Google Analytics as an EVENT hit:
+  - When users are shown the banner: `eventCategory = 'interventionBanner'`
+                                     `eventAction = 'interventionShown'`
+  - When users select the banner:    `eventCategory = 'interventionBanner'`
+                                     `eventAction = 'interventionClicked'`
+  - When users reject the banner:    `eventCategory = 'interventionBanner'`
+                                     `eventAction = 'interventionDismissed'`
+
+###### banner_feedback_results_page.sql
+
+- This script provides the number and proportion of sessions where the user has
+  left feedback about the quality of the results page of the checker, only if they
+  have accessed the checker via the banner intervention i.e. total number of
+  sessions that leave feedback on the results page / total number of sessions that
+  access the results page
+
+###### banner_feedback.sql
+
+- This scripts provides the number and proportion of sessions where the user has
+  left feedback about the quality of the page where the banner is shown i.e. total
+  number of sessions that leave feedback on the banner trigger page / total number
+  of sessions that are shown the banner.
+
+###### banner_journey_results_page.sql
+
+- This query extracts the total number of journeys that visit the results page (i.e.
+  completes the checker), and groups by the two SaB pagePaths that trigger the banner
+  intervention.
+- This will provide us with an understanding of which pagePaths are most most likely
+  to result in user engagement with the checker.
+
+###### banner_journey_results_link.sql
+
+- This query extracts the total number of journeys that select at least 1 link on
+  the results page, and groups by the two SaB pagePaths that trigger the banner
+  intervention.
+- This will provide us with an understanding of which pagePaths are most most likely
+  to result in user engagement with the checker.
+
+## Assumptions and caveats
+
+#### Nonequivalent group (email)
+
+- This group is defined by users that access the pagePath that starts with:
+`/next-steps-for-your-business?crn`. The query string `crn` represents users
+that have landed on the pagePath from the Companies House email.
+
+#### Experimental group (banner intervention)
+
+###### banner_feedback.sql
+
+- This script aims to identify sessions that leave feedback on the page the banner
+  intervention is shown.
+- This script calculates feedback sessions of interest as sessions that have an
+  EVENT hit `ffNoClick` or `ffYesClick` (i.e. provide feedback) immediately following
+  an EVENT hit `interactionShown` (i.e. shown the banner).
+- In addition, these two EVENT hits must have the same pagePath.
+- Therefore, it does not include sessions where other PAGE or EVENT hits occur
```

Comment:
May be clearer as
`- Therefore, it does not include sessions where other PAGE or EVENT hits occur
  between the EVENT hit `interactionShown` and the EVENT hit `ffNoClick` or `ffYesClick`.`